audit_log.py

```diff
@@ -0,0 +1,170 @@
+import socket
+import ssl
+import sys
+import argparse
+import ConfigParser
+import requests
+from jinja2 import Template
+import os
+import json
+import time
+import logging
+import logging.handlers
+from cb_defense_syslog import *
+import traceback
+import hashlib
+import fcntl
+
+
+#Get Audit Logs will return response
+def get_audit_logs(url, api_key_query, api_connector_id_query, ssl_verify, logger, proxies=None):
+    headers = {'X-Auth-Token': "{0}/{1}".format(api_key_query, api_connector_id_query)}
+    try:
+        response = requests.get("{0}/integrationServices/v3/auditlogs".format(url),
+                                headers=headers,
+                                timeout=15, proxies=proxies)
+
+        if response.status_code != 200:
+            logger.error("Could not retrieve audit logs: {0}".format(response.status_code))
+            return False
+
+        notifications = response.json()
+
+    except Exception as e:
+        logger.error("Exception {0} when retrieving audit logs".format(get_unicode_string(e)), exc_info=True)
+        return None
+
+    if notifications.get("success", False) != True:
+        logger.error("Unsuccessful HTTP response retrieving audit logs: {0}"
+                     .format(notifications.get("message")))
+        return False
+
+    notifications = notifications.get("notifications", [])
+    if not notifications:
+        logger.info("No audit logs available")
+        return False
+
+    return response
+
+
+#Parse responses. There are three options: cef, leef, json
```

Comment:
Any comment like this could go under the function as a doc string. I prefer the google style, http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings but any docstring is just fine.

Reply:
I removed those specific comments. I will be creating a version document that details the functionalities of the updated Syslog Connector .  If you feel I need to add more comments throughout the scripts, let me know and will be more than happy to do that!

Reply:
I created a ticket for creating comments in the Syslog Connector. Below is the Link: https://jira.carbonblack.local/browse/CBAPI-1212

Reply:
Works for me!